design/src/main/kotlin/uk/govuk/app/design/ui/component/Header.kt

```diff
@@ -40,27 +42,94 @@ fun TabHeader(
     }
 }
 
+data class ActionButton(
+    val text: String? = null,
+    val onClick: () -> Unit
+)
+
 @Composable
 fun ChildPageHeader(
-    text: String,
-    onBack: () -> Unit,
+    text: String? = null,
+    backButton: ActionButton? = null,
+    actionButton: ActionButton? = null,
     modifier: Modifier = Modifier,
 ) {
     Column(modifier) {
-        Box(
-            modifier = Modifier.height(64.dp),
-            contentAlignment = Alignment.CenterStart
-        ) {
-            TextButton(
-                onClick = onBack,
+        if (backButton != null || actionButton != null) {
+            Row(
+                modifier = Modifier
+                    .height(64.dp)
+                    .fillMaxWidth()
+                    .padding(GovUkTheme.spacing.small),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
             ) {
-                Icon(
-                    imageVector = Icons.AutoMirrored.Filled.ArrowBack,
-                    contentDescription = stringResource(R.string.content_desc_back),
-                    tint = GovUkTheme.colourScheme.textAndIcons.link
+                ChildPageHeaderBackButton(backButton, modifier = modifier.weight(1f))
```

Comment:
These changes appear to cause back button touch area to cover width of screen if no action button is needed and touch area cover half width of screen if action button is needed. 
![image](https://github.com/user-attachments/assets/749ca562-e898-497b-85e5-d917250263b0)
![image](https://github.com/user-attachments/assets/91af3803-c661-4651-805d-d4119f91b2bb)

Reply:
Think that's the `Spacer`'s - I had issues aligning the icons without them. I'll take another look 👀

Reply:
I believe this is now resolved - thanks @TimGJD